R/standalone-obj-type.R

```diff
@@ -311,10 +314,15 @@ stop_input_type <- function(x,
   if (length(what)) {
     what <- oxford_comma(what)
   }
+  if (inherits(arg, "AsIs")) {
```

Comment:
We are assuming that the `abort(arg = arg)` call down below can handle an `arg` that doesn't actually match any argument in the call by just ignoring it - because presumably if you supply an AsIs `arg` then it is going to look completely customized and won't match anything in the call.

Alternatively we could set something like `arg_highlight <- NULL` and pass that through to `abort()` in this case instead, but I will let Lionel make that decision